dev/examples/controls/line.js

```diff
@@ -533,8 +533,15 @@ export const renderLineY2AxisWithPanning = (id) => {
     const graphDataY2 = utils.deepClone(
         getDemoData(`#${id}`, "LINE_TIMESERIES").data[1]
     );
+    const graphDataY3 = {
+        ...graphDataY2,
+        values: [],
+        xlabel: "xlabel",
```

Comment:
Can you update the names of these variables based on our naming standards

xlabel -> xLabel
ylabel -> yLabel
y2label -> y2Label

Reply:
Updated [here](https://github.com/cerner/carbon-graphs/pull/290/commits/fe992a644e0629aab8eb64e8675de9b558eaddfd)